dockerfiles/bash/README.md

```diff
@@ -0,0 +1,59 @@
+# Experimental Bash language server 
+
+![](https://cl.ly/271p292i342p/Screen%20Recording%202018-05-07%20at%2009.19%20AM.gif)
+
+*This GIF was created browsing https://sourcegraph.com/github.com/creationix/nvm/-/blob/nvm.sh with this Bash language server enabled.*
+
+## Introduction
+
+This Dockerfile adds experimental Bash language support for Sourcegraph. 
+
+Thanks to the [mads-hartmann/bash-language-server](https://github.com/mads-hartmann/bash-language-server) project for providing the language server that's wrapped by `lsp-adapter` in this image.
+
+## How to add this language server to Sourcegraph
+
+*These steps are adapted from our documentation for [manually adding code intelligence to Sourcegraph server](https://about.sourcegraph.com/docs/code-intelligence/install-manual/).*
+
+1. Run the `sourcegarph/server` Docker image: 
+
+```shell
+docker run \
+  --publish 7080:7080 \
+  --rm \
+  --network=lsp \
+  --name=sourcegraph 
+  --volume ~/.sourcegraph/config:/etc/sourcegraph \
+  --volume ~/.sourcegraph/data:/var/opt/sourcegraph \
```

Comment:
Please think about multi-platform when providing shell commands - `\` is not an escape character in every shell, our docs don't use them for that reason